plugins/unlock

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
If this is a utility that will stick around, it needs copyright & license.

Reply:
Added

Reply:
confirmed